packages/backend-modules/payments/lib/services/UserService.ts

```diff
@@ -0,0 +1,35 @@
+import { PgDb } from 'pogi'
+import { UserRow } from '@orbiting/backend-modules-types'
+import { UserDataRepo, Address, AddressRow } from '../database/UserRepo'
+
+export class UserService {
+  protected pgdb: PgDb
+  protected users: UserDataRepo
+
+  constructor(pgdb: PgDb) {
+    this.pgdb = pgdb
+    this.users = new UserDataRepo(pgdb)
+  }
+
+  async updateUserName(
```

Comment:
where is this function used? It would be good to also update firstName and lastName on mailchimp if the name is updated